localization/languages/ar.json

```diff
@@ -13,6 +13,10 @@
         "saveImageAs": "حفظ الصورة باسم",
         "searchWith": "%s بحث مع", //%s will be replaced with the name of the current search engine
         "copyLink": "نسخ الرابط",
+        "select-all": "Select All", // Missing Translation
```

Comment:
Can you make these `null` instead, and for the other languages as well? localizationHelpers.js will use the English version automatically if there isn't a translated version available, and that way if we decide to change the strings later, we can do so without updating it in multiple places (although in this instance, changing the English strings does seem rather unlikely).

Reply:
Also, could this be `selectAll` to match how the other strings are named?